backend/schema/message.py

```diff
@@ -11,22 +13,58 @@ class Reaction(BaseModel):
     character: str
 
 
-class Thread(BaseModel):
+class ThreadRequest(BaseModel):
     """
     Provides a base model for all threads
     """
-
-    sender_id: str
-    room_id: str
     text: str
     reactions: List[Reaction] = []
     files: List[AnyHttpUrl] = []
     saved_by: List[str] = []
-    is_pinned: bool = False
-    is_edited: bool = False
     created_at: str = str(datetime.utcnow())
 
 
+class Thread(ThreadRequest):
```

Comment:
Docstring is missing.
Kindly update the class with a docstring.
`git commit `would not pass with the pre-commit config. @AI-fae

Reply:
Docstring added